libs/utils/android/workloads/uibench.py

```diff
@@ -85,7 +85,7 @@ def run(self, out_dir, test_name, duration_s, collect=''):
         self.out_dir = out_dir
         self.collect = collect
 
-        # Press Back button to be sure we run the video from the start
+        Screen.unlock(self._target)
         System.menu(self._target)
```

Comment:
Can you also please check if there are similar patterns in the other `libs/utils/android/workloads/*.py` and do the same change in all of them in this patch (which will became a more generic "android: use a more generic display unlocking function"